### PR TITLE
Delete note functionality

### PR DESCRIPTION
**Pull request by omkarpatke**
# Added Delete note functionality.

**Checks:**
- Notepage.js

### CODE REVIEW

File: src/Pages/NotesPage/NotesPage.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+.note-icon-paint:hover{
+    cursor: pointer;
+}
+
+ /* Note Cart */
+
+ .add-note-cart{
+     display: none;
+     z-index: 100;
```
Comment (JuvitaSaini09) [inline src/Pages/NotesPage/NotesPage.css:121]: you should avoid giving random z-index values

**Review by JuvitaSaini09 — APPROVED** [review]
your code looks good to merge

File: src/Pages/NotesPage/NotesPage.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+import { useNote } from '../../context/note-context';
+import { addNoteToBackend } from '../../api-calls/api-calls';
+import axios from 'axios';
+
+export default function NotesPage() {
+  const navigate = useNavigate();
+  const {notes , isLogin} = useNote();
+  const [showForm , setShowForm] = useState(false);
+  const [title , setTitle] = useState('');
+  const [desc , setDesc] = useState('');
```
Comment (yaswanthmyneni) [inline src/Pages/NotesPage/NotesPage.js:15]: suggestion: move these to the context

File: src/Pages/NotesPage/NotesPage.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+     }
+     
+   }
+
+
+   const deleteNote = async(note) => {
+     const encodedToken = localStorage.getItem('token');
+        try{ await axios.delete(`/api/notes/${note._id}` , {headers : {authorization : encodedToken}} , {note})}
+        catch (err) {
+          console.log(err)
```
Comment (yaswanthmyneni) [inline src/Pages/NotesPage/NotesPage.js:57]: suggestion: for displaying errors use console.error()

File: src/Pages/NotesPage/NotesPage.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+   const addNote = (e) => {
+     e.preventDefault();
+     if(isLogin){
+      const note = {
+        title,
+        desc,
+        time:new Date().toLocaleString(),
+      }
+      console.log(note)
```
Comment (yaswanthmyneni) [inline src/Pages/NotesPage/NotesPage.js:38]: changes: remove console logs when you push code to github

**Review by yaswanthmyneni — APPROVED** [review]
you can merge it.

**Review by Sahil25061999 — APPROVED** [review]
the code mostly looks good require some changes as suggested by yashwanth, there is typo on the signup page you are asking for full name instead of first name. rest looks good

**Pull request merged**